test/forked_joins.js

```diff
@@ -0,0 +1,168 @@
+var test = require('tape')
+var hyperlog = require('hyperlog')
+var fdstore = require('fd-chunk-store')
+var path = require('path')
+var memdb = require('memdb')
+var collect = require('collect-stream')
+
+var tmpdir = require('os').tmpdir()
+var storefile = path.join(tmpdir, 'osm-store-' + Math.random())
+
+var osmdb = require('../')
+var versions = {}
+var way0Key
+var osm = osmdb({
+  log: hyperlog(memdb(), { valueEncoding: 'json' }),
+  db: memdb(),
+  store: fdstore(4096, storefile)
+})
+
+test('setup db', function (t) {
```

Comment:
-1 to this style of testing: requires shared global mutable state, so tests are highly dependent on order and presence. I prefer contained, pure tests like [this](https://github.com/digidem/osm-p2p-db/blob/master/test/fork.js), where helper logic is broken out as (non-global statal) functions outside the test.